pkg/deploy/api/v1/types.go

```diff
@@ -111,8 +111,12 @@ const (
 // based on the specified container which is assumed to be part of the
 // deployment template.
 type ExecNewPodHook struct {
-	// Command is the action command and its arguments.
+	// Command represents an entrypoint array. Not executed within a shell. The docker image’s ENTRYPOINT
+	// is used if this is not provided.
 	Command []string `json:"command" protobuf:"bytes,1,rep,name=command"`
```

Comment:
@smarterclayton AFAIK we use only `Command` currently, so when you set the command in hook, it always become the entrypoint